### PR TITLE
cleanup(prep/v6-stable): spell-check and formatting for v6 stable

### PR DESCRIPTION
**Pull request by zakkarry**
final pass for jsdoc and spell-check and basic ide warnings for stable release v6

### CODE REVIEW

File: src/torznab.ts [hunk 1]
```diff
@@ -807,8 +807,7 @@ async function getAndLogIndexers(
 		);
 		if (!entry) return true;
 		if (entry.firstSearched < skipBefore) return false;
-		if (entry.lastSearched > skipAfter) return false;
-		return true;
+		return entry.lastSearched <= skipAfter;
```
Comment (ShanaryS) [inline src/torznab.ts:810]: The other got reverted as I was reviewing but I think the combined return is less intuitive.

**Review by ShanaryS — APPROVED** [review]

File: src/torznab.ts (hunk 1 above)
Comment (zakkarry) [reply src/torznab.ts]: Sure, reverted this one too.

**Review by ShanaryS — APPROVED** [review]

**Comment by NLZ** [comment]
Could we please delete these empty lines between the comment and the config item?

https://github.com/cross-seed/cross-seed/blob/0879c149d6e646a91ea91abf93640cdbb2357b0a/src/config.template.cjs#L127-L129

https://github.com/cross-seed/cross-seed/blob/0879c149d6e646a91ea91abf93640cdbb2357b0a/src/config.template.cjs#L146-L148

**Comment by zakkarry** [comment]
> Could we please delete these empty lines between the comment and the config item?
> 
> https://github.com/cross-seed/cross-seed/blob/0879c149d6e646a91ea91abf93640cdbb2357b0a/src/config.template.cjs#L127-L129
> 
> https://github.com/cross-seed/cross-seed/blob/0879c149d6e646a91ea91abf93640cdbb2357b0a/src/config.template.cjs#L146-L148

Just remove them from your config if you don't want them there?

**Comment by NLZ** [comment]
It is to prettify the config to make it uniform, as these two config entries are different from the others which don't have empty line between the comment and the config.

**Comment by zakkarry** [comment]
> It is to prettify the config to make it uniform, as these two config entries are different from the others which don't have empty line between the comment and the config.

So open a PR? This one was merged before you even commented...

**Comment by zakkarry** [comment]
> It is to prettify the config to make it uniform, as these two config entries are different from the others which don't have empty line between the comment and the config.

https://github.com/cross-seed/cross-seed/pull/809/commits/7616c2e4a703bedfc77b528eb5a161cf18bbc9d4

**Pull request merged**